config/locales/fil/splash.fil.yml

```diff
@@ -0,0 +1,11 @@
+fil:
+  pages:
+    index:
+      life-is-tough: 'Minsan ang buhay ay matigas ...'
+      find-refuge: 'Maghanap ng Refuge'
+      add-restroom-button-label: 'Magdagdag ng isang Silid'
```

Comment:
Not sure if it comes across clearly that this means a restroom/bathroom/toilet, rather than a generic "room" potentially without a toilet?

(Hopefully this is my last review comment! Sorry not noticing these things until now.)

Reply:
Hi, sorry I just noticed this now been busy yesterday but I've basically used the word `banyo` from our previous conversations. Thanks for noticing this!